### PR TITLE
New feature added for manual testing AB#18557

### PR DESCRIPTION
**Pull request by girtsfre**
Created a feature file with test scenarios for testing in Swagger

### CODE REVIEW

**Review by TatevAccenture — DISMISSED** [review]

File: features/Other/Edit_Info_Person_Affected_From_EspoCRM.feature [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Background:
+Given a user in the Swagger UI
+
+Scenario: Successfully Update FSP from Whatsapp to Jumbo from EspoCRM
+
+    Given the "programId" is provided
+    Given the "refferenceId" is provided
+    Given the "NewFspAttributes" is provided
+    When 'newFspName' value is entered as 'Intersolve-jumbo-physical'
+    And the user has filled in values for "NewFspAttributes"
```
Comment (jannisvisser) [inline features/Other/Edit_Info_Person_Affected_From_EspoCRM.feature:13]: I believe the idea here is that there can only 1 'When' statement (and thus that there cannot be 'And's after a 'When'), as the 'When' should really pinpoint 1 specific action that is done by the user in a scenario. So all 'preparation' before this action should be under 'Given'. (Same for the other scenarios below.)

File: features/Other/Edit_Info_Person_Affected_From_EspoCRM.feature [hunk 2]
```diff
@@ -0,0 +1,35 @@
+@swagger-ui
+Feature: Edit info From ESPO CRM
+
+Background:
+Given a user in the Swagger UI
+
+Scenario: Successfully Update FSP from Whatsapp to Jumbo from EspoCRM
+
+    Given the "programId" is provided
+    Given the "refferenceId" is provided
```
Comment (jannisvisser) [inline features/Other/Edit_Info_Person_Affected_From_EspoCRM.feature:10]:
```suggestion
    Given the "referenceId" is provided
```

Comment (jannisvisser) [reply features/Other/Edit_Info_Person_Affected_From_EspoCRM.feature:10]: same change for other scenarios below

**Review by jannisvisser — CHANGES_REQUESTED** [review]
these are just some minor changes. if they're changed, i'll approve it.

**Review by jannisvisser — APPROVED** [review]

**Pull request merged**